plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/command/edit/EditCommandsViewImpl.java

```diff
@@ -333,26 +353,19 @@ public void onClick(ClickEvent event) {
                                                         delegate.onCloseClicked();
                                                     }
                                                 });
-
-        saveButton = createButton(coreLocale.save(), "window-edit-configurations-save", new ClickHandler() {
+        closeButton.addDomHandler(new BlurHandler() {
             @Override
-            public void onClick(ClickEvent event) {
-                delegate.onSaveClicked();
+            public void onBlur(BlurEvent blurEvent) {
+                //set default focus
+                selectElementText(filterInputField.getElement());
             }
-        });
-        saveButton.addStyleName(resources.windowCss().primaryButton());
-
-        cancelButton = createButton(coreLocale.cancel(), "window-edit-configurations-cancel", new ClickHandler() {
-            @Override
-            public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
-            }
-        });
-
-        overFooter.add(saveButton);
-        overFooter.add(cancelButton);
+        }, BlurEvent.getType());
 
         addButtonToFooter(closeButton);
+
+        Element dummyFocusElement = DOM.createSpan();
```

Comment:
This dummy focus element allows us to keep the focus on the current popup(after click tab on the last focus element of current popup).